config/secrets.yml

```diff
@@ -75,12 +169,11 @@ production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   sendgrid: <%= !ENV["SENDGRID_USERNAME"].blank? %>
   smtp_username: <%= ENV["MAILER_SMTP_USER_NAME"] || ENV["SENDGRID_USERNAME"] %>
-  smtp_authentication_username: <%= ENV["MAILER_SMTP_AUTHENTICATION_USER_NAME"] %>
```

Comment:
👀

Reply:
It has been renamed to be the same as in the clean app